packages/cli/src/models/initializer/truffle/Truffle.ts

```diff
@@ -52,6 +52,23 @@ const Truffle = {
     }
   },
 
+  getContractNames(): string[] {
+    const buildDir = this.getBuildDir();
+    if (buildDir) {
+      return FileSystem.readDir(buildDir)
+        .filter((name) => {
```

Comment:
We can manually filter for json extensions, or use [glob](https://www.npmjs.com/package/glob) to do so